compiler/rustc_middle/src/mir/interpret/value.rs

```diff
@@ -28,16 +26,25 @@ pub struct ConstAlloc<'tcx> {
     pub ty: Ty<'tcx>,
 }
 
+#[derive(Copy, Clone, Debug, Eq, PartialEq, Hash)]
+#[derive(HashStable)]
+pub struct ConstValue<'tcx>(pub(crate) Interned<'tcx, ConstValueKind<'tcx>>);
+
 /// Represents a constant value in Rust. `Scalar` and `Slice` are optimizations for
 /// array length computations, enum discriminants and the pattern matching logic.
 #[derive(Copy, Clone, Debug, Eq, PartialEq, TyEncodable, TyDecodable, Hash)]
 #[derive(HashStable, Lift)]
-pub enum ConstValue<'tcx> {
+pub enum ConstValueKind<'tcx> {
     /// Used for types with `layout::abi::Scalar` ABI.
     ///
     /// Not using the enum `Value` to encode that this must not be `Uninit`.
```

Comment:
I do wonder which enum this refers to... I think it's just a very outdated comment.^^